api/src/main/java/org/openmrs/PatientIdentifier.java

```diff
@@ -204,17 +204,6 @@ public void setPreferred(Boolean preferred) {
 		this.preferred = preferred;
 	}
 	
-	/**
-	 * @return the preferred status
-	 * 
-	 * @deprecated as of 2.0, use {@link #getPreferred()}
-	 */
```

Comment:
We do not deprecate and remove at the same time. After deprecation, we still leave it in for a number of versions to come until when we eventually remove it.

Reply:
@dkayiwa Fixed :)